### ISSUE TITLE
Blank page on resend - PHP Error 

### ISSUE THREAD

**Issue by AntonioCS** [CalderaWP/Caldera-Forms#2185]
Caldera form v 1.5.7.1
Wordpress v 4.9.1 

On page 
/wp-admin/admin.php?page=caldera-forms&message_resent=true 

I click on "Entries" under "Invite Friends". I see a list of the email on my right side.
When I click resend I see a blanck page and on my docker debug window I see the following:

```
site-php-fpm | NOTICE: PHP message: PHP Fatal error:  Uncaught TypeError: Argument 2 passed to Caldera_Forms_Field_Util::has_field_type() must be of the type array, null given, called in /application/wp-content/plugins/caldera-forms/classes/magic.php on line 49 and defined in /application/wp-content/plugins/caldera-forms/classes/field/util.php:361
site-php-fpm | Stack trace:
site-php-fpm | #0 /application/wp-content/plugins/caldera-forms/classes/magic.php(49): Caldera_Forms_Field_Util::has_field_type('credit_card_exp', NULL)
site-php-fpm | #1 /application/wp-includes/class-wp-hook.php(286): Caldera_Forms_Magic->field_magic(NULL, '%person_one_ema...', Array, NULL, NULL)
site-php-fpm | #2 /application/wp-includes/plugin.php(203): WP_Hook->apply_filters(NULL, Array)
site-php-fpm | #3 /application/wp-content/plugins/caldera-forms/classes/magic/doer.php(57): apply_filters('caldera_forms_p...', NULL, '%person_one_ema...', Array, NULL, NULL)
site-php-fpm | #4 /application/wp-content/plugins/caldera-forms/classes/core.php(2029): Caldera_Forms_Magic_Doer::do_field_magic('%person_one_ema...', NULL, NULL)
site-php-fpm | #5 /applicati...
site-php-fpm | 172.19.0.2 -  01/Jan/2018:13:52:04 +0000 "GET /wp-admin/index.php" 500
site-webserver | 2018/01/01 13:52:05 [error] 5#5: *254 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught TypeError: Argument 2 passed to Caldera_Forms_Field_Util::has_field_type() must be of the type array, null given, called in /application/wp-content/plugins/caldera-forms/classes/magic.php on line 49 and defined in /application/wp-content/plugins/caldera-forms/classes/field/util.php:361
site-webserver | Stack trace:
site-webserver | #0 /application/wp-content/plugins/caldera-forms/classes/magic.php(49): Caldera_Forms_Field_Util::has_field_type('credit_card_exp', NULL)
site-webserver | #1 /application/wp-includes/class-wp-hook.php(286): Caldera_Forms_Magic->field_magic(NULL, '%person_one_ema...', Array, NULL, NULL)
site-webserver | #2 /application/wp-includes/plugin.php(203): WP_Hook->apply_filters(NULL, Array)
site-webserver | #3 /application/wp-content/plugins/caldera-forms/classes/magic/doer.php(57): apply_filters('caldera_forms_p...', NULL, '%person_one_ema...', Array, NULL, NULL)
site-webserver | #4 /application/wp-content/plugins/caldera-forms/classes/core.php(2029): Caldera_Forms_Magic_Doer::do_field_magic('%person_one_ema...', NULL, NULL)
```

I am getting this on my local machine and on an ec2 instance. 
The email are also not being sent (that is why I am trying to resend them via this)

As you can see this is a PHP error.
I tried fixing this by doing $form = []; but I got more errors in other scripts.

Please fix this.

**Comment by Shelob9** [comment]
@AntonioCS 

Thanks for reporting this. Do you get the same error in Caldera Forms 1.5.8 ?

This should have been fixed with  #1864

**Comment by AntonioCS** [comment]
We needed this to work so we changed plugins.
We got the plugin like 2 days ago. How can a bug like this make it into production?

Thanks for the help but like I said we moved on.
